app/views/admin/facilities/index.html.erb

```diff
@@ -8,7 +8,7 @@
 
     <% if current_admin.accessible_organizations(:manage).any? || current_admin.power_user? %>
       <%= link_to new_admin_facility_group_path, class: "btn btn-sm btn-success" do %>
-        <i class="fas fa-plus mr-1"></i> Facility group
+        <i class="fas fa-plus mr-1"></i> <%= t("facility_group").capitalize %>
```

Comment:
@timcheadle this would be clearer if it said "Add" before the name of the facility group. Example "+ Add Upazila" instead of "+ Upazila".

Reply:
Plus means "Add". Let's keep this simple and consistent. "+ District" is good.

Reply:
@dburka it's not consistent in other places, and "+ District" is not as clear as "+ Add district". Should we also update the "Add a facility" button to say "+ facility"?

Reply:
Yes. I just updated the "+ Facility" button as well.